spec/factories/glossaries.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :glossary, class: Glossary do
```

Comment:
Where the class and the factory name are the same, you do not need the `class: Glossary` syntax. FactoryBot is smart enough to detect this itself.

```suggestion
  factory :glossary do
```